### ISSUE TITLE
How do I look up a place by its address?

### ISSUE THREAD

**Issue by dave-kennedy** [organicmaps/organicmaps#1238]
When I try to look up "angelos cds", a couple search results show up but not the one I'm looking for. That's okay, I figure nobody has contributed this particular location to OSM. I know it's address so I try to look up "1959 s broadway denver co 80210". Now the app shows several streets and a couple hamlets, but not the address I specified. Now, I figure it must be a OSM issue. However, when I look up the address in OSM it comes right up: https://www.openstreetmap.org/search?query=1959%20s%20broadway%20denver%20co%2080210#map=19/39.68109/-104.98768

Am I doing something wrong or is looking up a place by its address unsupported?

**Comment by biodranik** [comment]
You're doing everything right. There is a bug somewhere with "s" => "south", we'll fix it. Does the search work if you type "south" instead of "s"?

**Comment by dave-kennedy** [comment]
No difference, unfortunately:

![Screenshot_20210905-102417_Organic_Maps](https://user-images.githubusercontent.com/13735532/132134268-7dd14b0f-9cab-4eed-8fda-576e5cc6adf0.png)

**Comment by bnitkin** [comment]
This looks like the same issue I was running into in https://github.com/organicmaps/organicmaps/issues/1120#issuecomment-913183527 - OSM's Nominatim merges a few different sources when searching, but Organic Maps only uses the OSM data.

On the map, you can see 1951 S Broadway, but not 1959. It's pretty easy to add new addresses & buildings (click edit, make an account, go through the tutorial, and start editing). Not certain how long edits take to show up in Organic Maps.

Maybe a different direction for search - would it be possible to "fuzz" street addresses? If 1959 S Broadway isn't found but 1951 is, that might be a close-enough suggestion for people.

**Comment by biodranik** [comment]
Right. There is no building with that address in OSM, that's why our search fails to find it.

**Comment by keunes** [comment]
Related: #1263

**Comment by vng** [comment]
UPD: "1959 s broadway denver" works after integrating Tiger.
